src/test/groovy/com/ofg/uptodate/finder/HttpProxyServerProvider.groovy

```diff
@@ -0,0 +1,29 @@
+package com.ofg.uptodate.finder
+
+import com.github.tomakehurst.wiremock.client.MappingBuilder
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.ofg.uptodate.http.HttpMockServer
+
+class HttpProxyServerProvider {
```

Comment:
@PackageScope added, cannot add @CompileStatic because of the Groovy version used in the plugin.